### ISSUE TITLE
Empty java.util.PriorityQueue causes IllegalArgumentException when loaded

### ISSUE THREAD

**Issue by fh-ms** [microstream-one/microstream#233]
## Environment Details
* MicroStream Version: 05.00.02
* JDK version: all
* OS: all

## Describe the bug
Bug in type handler for java.util.PriorityQueue. 
An exception is thrown when an empty queue is loaded.
```
java.lang.IllegalArgumentException
	at java.util.PriorityQueue.<init>(PriorityQueue.java:168)
	at one.microstream.persistence.binary.java.util.BinaryHandlerPriorityQueue.create(BinaryHandlerPriorityQueue.java:130)
	at one.microstream.persistence.binary.java.util.BinaryHandlerPriorityQueue.create(BinaryHandlerPriorityQueue.java:1)
	at one.microstream.persistence.binary.internal.AbstractBinaryHandlerCustom.create(AbstractBinaryHandlerCustom.java:1)
	at one.microstream.persistence.binary.types.BinaryLoader$Default.createBuildItem(BinaryLoader.java:221)
	at one.microstream.persistence.binary.types.BinaryLoader$Default.internalReadBinaryEntities(BinaryLoader.java:174)
	at one.microstream.persistence.binary.types.BinaryLoader$Default.readBinaryEntities(BinaryLoader.java:162)
	at one.microstream.persistence.binary.types.ChunksBuffer.iterateEntityDataLocal(ChunksBuffer.java:286)
	at one.microstream.persistence.binary.types.ChunksBuffer.iterateEntityData(ChunksBuffer.java:295)
	at one.microstream.persistence.binary.types.BinaryLoader$Default.addChunks(BinaryLoader.java:820)
	at one.microstream.persistence.binary.types.BinaryLoader$Default.readLoadOidData(BinaryLoader.java:790)
	at one.microstream.persistence.binary.types.BinaryLoader$Default.readLoadOnce(BinaryLoader.java:732)
	at one.microstream.persistence.binary.types.BinaryLoader$Default.get(BinaryLoader.java:843)
	at one.microstream.persistence.binary.types.BinaryLoader$Default.loadRoots(BinaryLoader.java:888)
	at one.microstream.storage.embedded.types.EmbeddedStorageManager$Default.loadExistingRoots(EmbeddedStorageManager.java:340)
	at one.microstream.storage.embedded.types.EmbeddedStorageManager$Default.initialize(EmbeddedStorageManager.java:359)
	at one.microstream.storage.embedded.types.EmbeddedStorageManager$Default.start(EmbeddedStorageManager.java:248)
	at one.microstream.storage.embedded.types.EmbeddedStorageManager$Default.start(EmbeddedStorageManager.java:1)
	at one.microstream.storage.embedded.types.EmbeddedStorage.createAndStartStorageManager(EmbeddedStorage.java:651)
	at one.microstream.storage.embedded.types.EmbeddedStorage.start(EmbeddedStorage.java:508)
	at test.microstream.handler.BinaryHandlerTest.saveAndLoadEmptyTest(BinaryHandlerTest.java:40)
```

Include stack traces or command outputs

## To Reproduce
Just save an instance without data of java.util.PriorityQueue and restart the storage.

**Comment by fh-ms** [comment]
Fixed by #234

**Comment by zdenek-jonas** [comment]
Ok, fixed
